prboom2/src/gl_main.c

```diff
@@ -448,25 +448,37 @@ void gld_MapDrawSubsectors(player_t *plr, int fx, int fy, fixed_t mx, fixed_t my
   dsda_GLSetScreenSpaceScissor(fx, fy, fw, fh);
   glEnable(GL_SCISSOR_TEST);
 
+  // Look "down" at the floor, since subsector flats extend along x and z coordinates
+  glRotatef(-90.0, 1.0f, 0.0f, 0.0f);
+
   if (automap_rotate)
   {
     float pivotx = (float)(fx + fw / 2);
     float pivoty = (float)(fy + fh / 2);
 
-    float rot = -(float)(ANG90 - viewangle) / (float)(1u << 31) * 180.0f;
+    float rot = (float)(ANG90 - viewangle) / (float)(1u << 31) * 180.0f;
 
     // Apply the automap's rotation to the vertexes.
-    glTranslatef(pivotx, pivoty, 0.0f);
-    glRotatef(rot, 0.0f, 0.0f, 1.0f);
-    glTranslatef(-pivotx, -pivoty, 0.0f);
+    glTranslatef(pivotx, 0.0f, pivoty);
```

Comment:
Just want to double check, do the changes to the y/z axes here require any changes in the transforms below? (https://github.com/kraflab/dsda-doom/pull/288/files#diff-9467dec909417e0c3a69bffbdd2b00caac451893686c9f078d619bcd054e2fe4R535-R537)

Reply:
Double checked this stuff in my test map with scaled, translated, and rotated flats, all looks good.